3.component_model.md

```diff
@@ -258,14 +258,16 @@ Resources describe compute resources attached to a runtime.
 
 | Attribute | Type | Required | Default Value | Description |
 |-----------|------|----------|---------------|-------------|
-| `cpu` | [`CPU`](#cpu) | Y |  | Specifies the attributes of the cpu resource required for the container. |
-| `memory` | [`Memory`](#memory) | Y | | Specifies the attributes of the memory resource required for the container. |
+| `cpu` | [`CPU`](#cpu) | N |  | Specifies the attributes of the cpu resource required for the container. |
+| `memory` | [`Memory`](#memory) | N | | Specifies the attributes of the memory resource required for the container. |
```

Comment:
I think that making platform assume defaults for CPU and Memory can lead to difficult to debug container activation issues for the application operator.

Reply:
Also, if we allow relying on platforms defaults, then we should add a way by which the operator can specify resources in the application configuration so that the component can be deployed to a platform whose defaults are too low for that component.

---
In reply to: [335190832](https://github.com/oam-dev/spec/pull/218#discussion_r335190832) [](ancestors = 335190832)